src/frontend/sider/Fagsak/Behandling/context/BehandlingContext.tsx

```diff
@@ -271,23 +246,23 @@ export const BehandlingProvider = ({ children }: React.PropsWithChildren) => {
                 settIkkeKontrollerteSiderTilManglerKontroll,
                 søkersMålform,
                 trinnPåBehandling,
-                åpenBehandling,
+                behandling: behandling,
                 opprettBehandling,
                 logg,
                 hentLogg,
                 behandlingsstegSubmitressurs,
                 vilkårsvurderingNesteOnClick,
                 behandlingresultatNesteOnClick,
-                settÅpenBehandling,
                 oppdaterRegisteropplysninger,
                 foreslåVedtakNesteOnClick,
-                behandlingPåVent: hentDataFraRessurs(åpenBehandling)?.behandlingPåVent,
+                behandlingPåVent: behandling?.behandlingPåVent,
```

Comment:
Tror kanskje ikke du trenger `?` her 

```suggestion
                behandlingPåVent: behandling.behandlingPåVent,
```